compositor_render/src/renderer/format/yuv_to_rgba.wgsl

```diff
@@ -1,11 +1,13 @@
 struct VertexInput {
     @location(0) position: vec3<f32>,
     @location(1) tex_coords: vec2<f32>,
+    @location(2) video_id: i32,
```

Comment:
you are trying too much to force too much functionality on a single abstraction. video_id or input_id does not make sense in the color converter. To me, it's a sign that maybe shaders and color converters should have a separate pipeline.